### DISCUSSION TITLE
from where to download the twelve_spp_proteins.tar.gz file

### DISCUSSION THREAD

**Discussion by pjyoti29** [hahnlab/CAFE5#93] [Tutorial] [answered]
Hi..
I am new to CAFE analysis. I am trying with the tutorial plz help me getting the "twelve_spp_proteins.tar.gz" file.

Thanks and Regards

**Comment by ulaszewski** [comment]
Hi @pjyoti29 ,
It seems that the tutorial as well as the files are not maintained at any sites at the moment, luckaly I've downloaded the whole tutorial around two years ago. 
I've uploaeded it to my personal Google Drive as well as the twelve_spp_proteins.tar.gz file:
https://drive.google.com/drive/folders/1SMDE_2SeMB_NC1oG01SbghmwD0LCMrvy?usp=sharing
Hope it helps

**Comment by pjyoti29** [comment]
Thank you so much. This will be highly helpful to learn CAFE.



On Tue, Oct 11, 2022 at 2:16 PM Bartosz Ulaszewski ***@***.***>
wrote:

> Hi @pjyoti29 <https://github.com/pjyoti29> ,
> It seems that the tutorial as well as the files are not maintained at any
> sites at the moment, luckaly I've downloaded the whole tutorial around two
> years ago.
> I've uploaeded it to my personal Google Drive as well as the
> twelve_spp_proteins.tar.gz file:
>
> https://drive.google.com/drive/folders/1SMDE_2SeMB_NC1oG01SbghmwD0LCMrvy?usp=sharing
> Hope it helps
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/hahnlab/CAFE5/issues/84#issuecomment-1274329299>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ANE6PSJEYACOCJBWXJWVN3TWCUSOVANCNFSM6AAAAAAQNAV34A>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
Dr. JYOTI PATHAK
Research Associate
ICAR-National Bureau of Agricultural Insect Resources
Hebbal, Bangalore

**Comment by LeonidasCl** [comment]
您好，来信已收到，我会尽快回复，急事请来电。（这是一封自动回复邮件）

**Comment by benfulton** [comment, marked as answer]
There is now a new home for these files. They are downloadable from

https://datacore.iu.edu/concern/data_sets/6m311q02z?locale=en

The tutorial files themselves are at

https://github.com/hahnlab/cafe_tutorial

**Comment by LeonidasCl** [comment]
您好，来信已收到，我会尽快回复，急事请来电。（这是一封自动回复邮件）
